src/constants/devices.js

```diff
@@ -0,0 +1,13 @@
+const size = {
+  mobile: "375px",
```

Comment:
disredgard the earlier comment, you can just use template strings to interpolate:
```
... with other variables
name: `calc(${minFontSize} + (30 - 16) * ((100vw - ${maxFontSize}px) / ( ${desktopBreakpoint} - ${mobileBreakpoint})))`
...
```